### PR TITLE
add vercel challenge & guide

### PR DESCRIPTION
**Pull request by CJAkkuec**
> Solves #640

- added an extra challenge to the challenge document about deploying to Vercel
- included a step-by-step guide

Additional Tasks:
- add a reminder referencing the guide somewhere in one of the Capstone sessions?

### CODE REVIEW

**Review by ahohnsen — CHANGES_REQUESTED** [review]
Hey CJ,

Thank you for handling this ticket. :) 

As I mentioned in my comments, I recommend adding a section on how to disable the deployment protection. This will ensure the deployment remains accessible.

File: sessions/recap-project-1/quiz-app/README.md [hunk 1]
```diff
@@ -122,6 +122,18 @@ During **"Recap Project 1"** you can essentially "skip" this step, because you'r
...
+> - navigating to the "Settings" tab of your project on Vercel,
+> - selecting "Deployment Protection" and
+> - clicking on the "Vercel Authentication" toggle.
+>
+> This can also be accomplished after the project has already been deployed.
+>
+> | Step     | Screen                        |
+> | -------- | ----------------------------- |
+> | Settings | ![Add](./assets/screen11.png) |
+
```
Comment (ahohnsen) [inline sessions/recap-project-1/quiz-app/README.md:136]: In my account, I can only adjust that setting after deployment, not before. I'm not sure if this is always the case. If so, I would suggest moving this section to the bottom.

**Review by ahohnsen — APPROVED** [review]
Very nice! 

Just one comment... :)

File: sessions/recap-project-1/quiz-app/README.md (hunk 1 above)
Comment (CJAkkuec) [reply sessions/recap-project-1/quiz-app/README.md:136]: True!

**Pull request merged**